plugins/janus_nosip.c

```diff
@@ -844,6 +844,22 @@ void janus_nosip_create_session(janus_plugin_session *handle, int *error) {
 	session->media.pipefd[0] = -1;
 	session->media.pipefd[1] = -1;
 	session->media.updated = FALSE;
+	session->media.audio_srtp_in = NULL;
+	session->media.audio_srtp_out = NULL;
+	session->media.audio_remote_policy.key = NULL;
+	session->media.audio_remote_policy.ssrc.type = ssrc_any_inbound;
+	session->media.audio_remote_policy.next = NULL;
+	session->media.audio_local_policy.key = NULL;
+	session->media.audio_local_policy.ssrc.type = ssrc_any_inbound;
+	session->media.audio_local_policy.next = NULL;
+	session->media.video_srtp_in = NULL;
+	session->media.video_srtp_out = NULL;
+	session->media.video_remote_policy.key = NULL;
+	session->media.video_remote_policy.ssrc.type = ssrc_any_inbound;
+	session->media.video_remote_policy.next = NULL;
+	session->media.video_local_policy.key = NULL;
+	session->media.video_local_policy.ssrc.type = ssrc_any_inbound;
+	session->media.video_local_policy.next = NULL;
```

Comment:
There's no need for any of that (apart maybe for the `ssrc_any_inbound`), we allocate memory with `g_malloc0` that sets everything to `0` and `NULL` already.